### ISSUE TITLE
Way of committing to repo

### ISSUE THREAD

**Issue by dselivanov** [ropensci/tokenizers#45]
I'm glad to see that `tokenizers` emerges to a common block for different packages. Also I see great discussion here - #25. 
What is **critical** in my opinion is to ensure all stakeholders understand how API changes with every commit which introduce additional fields in output or if format changes (and can break dependent packages). So I want to ask to **discuss such changes in pull requests** which brings them. In that case every involved party can understand what is happening. It will be much easier to follow, comment and approve.

Long story short, let's commit significant changes only via pull requests (I believe it is not hard and won't slow down development).

**Comment by lmullen** [comment]
@dselivanov Yes, that's a good suggestion. I agree that for any major changes and certainly for all breaking changes that there should be a PR to allow for the discussion. 

I have a few changes I need to make for v0.2 but they aren't breaking, but once development starts on v0.3 we should all follow this model.

**Comment by Ironholds** [comment]
That sounds good to me. The API should be expected to be unstable <1.0.0 from a user POV, but agreement within the dev community is key.
